### PR TITLE
Shard conformance suites based on variation timing

### PR DESCRIPTION
**Pull request by aaroncameron-wk**
#### Reason for change
Some conformance suites have only a few long-running test cases that can't be divided into shards based on current logic.

#### Description of change
Previous approach: Generate temporary testcase indexes for each shard that specify which testcases that shard runs.
Proposed approach: Use CLI argument to filter test case variations based on ID. Variations that are filtered out in each shard get a "skip" pytest result in that shard's results.

Note that as a part of this, I've removed the `expected_empty_testcases` config, as it was causing issues and did not seem necessary once we were handling things at a variation level.

#### Steps to Test
CI

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: arelle/CntlrCmdLine.py
Comment (austinmatherne-wk) [inline arelle/CntlrCmdLine.py:292]: arbitrary line:

Nice work! Given that we have access to more nodes now, how do you feel about removing the double approval requirement for conformance suite runs? I know I'm tired of waiting for find tests to complete just to approve a workflow a second time 😅

**Review by austinmatherne-wk — DISMISSED** [review]

**Review by austinmatherne-wk — APPROVED** [review]

**Pull request merged**
